cmake/BLTPrivateMacros.cmake

```diff
@@ -389,6 +389,18 @@ macro(blt_setup_target)
             blt_add_target_link_flags(TO ${arg_NAME}
                                       FLAGS ${_BLT_${uppercase_dependency}_LINK_FLAGS} )
         endif()
+        # Propagate the overridden linker language, if applicable
+        if(TARGET ${dependency})
+            # If it's an interface library CMake doesn't even allow us to query the property
+            get_target_property(_dep_type ${dependency} TYPE)
+            if(NOT "${_dep_type}" STREQUAL "INTERFACE_LIBRARY")
```

Comment:
This is a complete shot in the dark.. but did you try INTERFACE_BLT_LINKER_LANGUAGE_OVERRIDE? maybe they only whitelist based on INTERFACE_?

Reply:
This does work (I was surprised!) but I'm having trouble thinking of a situation where we'd want to query interface libraries for this property, since it really only applies to library targets created by the current project.

Reply:
Does it also auto propagate via CMake? Even if it doesnt it does continue the current CMake convention that INTERFACE properties propagate to their parents...

Reply:
It does not, but I think that is still a better name for it.